third_party/silabs/silabs_executable.gni

```diff
@@ -68,7 +68,7 @@ template("silabs_executable") {
   ]
   copy(flashing_runtime_target) {
     sources = flashing_script_inputs
-    outputs = [ "${root_out_dir}/{{source_file_part}}" ]
+    outputs = [ "${root_out_dir}/${target_name}--{{source_file_part}}" ]
```

Comment:
I believe the "copy" action only needs to be done once, however now that `silabs_executable` is being called many times it ends up trying to copy these same python files repeatedly which causes an error.  I could not find a clean way of making it only do the copy once, so as a temporary fix I changed the names of the files to be something unique.  However ultimately this is not what we want, so I'll need to circle back and fix this properly.